README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Stock-Metric
 
-## Available Scripts
+> We built a webpage with React & Redux where you can be up to date on the stock market via Financial Modelling API.
 
-In the project directory, you can run:
+### Live GitHub Page
+[Live](https://stometric.netlify.app/)
```

Comment:
- [ ] It will be great to ensure that the live version works as it should. The app is properly deployed however, it seems there is an issue with `cors` since the app can not retrieve the data. Kindly check the error am getting ⬇️ 

![liveversion](https://user-images.githubusercontent.com/32361963/205417220-29ad3ec4-d550-4721-a38a-fcf828539464.png)

Reply:
@peterrobert Hello, Peter. Can it be that it is a problem in your browser? I have tried it in more than one computer and didn't have any errors about the CORS policy. The issue that I am having is about the suscription to the API as I have reached the maximum fetch on days.

Reply:
@sntgopl  i have tried it with a different browser and i still have the same issue. Which browser are you using?

Reply:
@peterrobert Googlechrome. The thing is it was just working, but with this API, the ones that Microverse gave us we only have 100 fetchs a day and I'm quite sure I reached the limit. Could you try again in half an hour? That's supposed when the day passes in my time zone and the API should be up already. Or rate the project on the video presentation I did in loom. I'll leave the link here:
https://www.loom.com/share/98f08e7db5704e9f8c7de287dbf54c86
Thank you, please let me know anything I can do to solve this issue.

Reply:
Okay, I understand. I cannot use the video presentation but we can wait for another 30 mins before we review it again. Just submit your project for another review once you confirm that the fetching of the data works perfectly. I will be happy to review your project then 😃

Reply:
Ok. Sounds great. Thank you peter! @peterrobert

Reply:
@peterrobert It seems cooldown it's refreshing in 24 hours. I'm not risking the time on approval so I changed the whole API to one in CRYPTO. I'll be submitting again the project soon.